src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
You can remove the `else` keyword here since the `if` block contains a `return` statement. This will make the code cleaner and easier to read, as described in checklist items 4 and 5.